spec/migration_spec.rb

```diff
@@ -6,22 +6,22 @@
   before(:each) do
     define_schema do
 
-      create_table :users do |t|
-        t.string :login, :index => { :unique => true }
+      create_table :users, id: :integer do |t|
```

Comment:
Instead of doing this, change the column types of the FKs to bigint.

Reply:
That leads to a much bigger diff, as you need to change the type to `:bigint` everywhere -- see teamsimplepay/schema_auto_foreign_keys@9601958fbf73a12d5019f85cc50aff1ae439f660. I prefer the current approach, it's explicit and unlikely to break on Rails upgrades.

Reply:
I'd rather include the bigtint change (as this is what I am doing with all the other gems)

Reply:
And once this is in, we can create a new PR to get it supporting AR 6.1 & 7 (I have PRs for most of the other gems mostly done)

Reply:
Pushed the changes, as requested :+1: